frontend/conf/application.conf

```diff
@@ -87,6 +89,10 @@ touchpoint.backend {
 
                 friend = ""
                 staff = ""
+                supporter {
+                    monthly = ""
+                    annual = ""
+                }
```

Comment:
I've added the relevant properties in private configs for supporter. 
I assume the facebooke and google stuff can be added in here.